### ISSUE TITLE
[Question] Clarification regarding Encoder training strategy in Appendix A.2

### ISSUE THREAD

**Issue by ZhouJankin** [yyfz/Pi3#97]
Hi, thank you for your impressive work on $\pi^3$.

I am reading the Training Details in Appendix A.2 and noticed a contradiction regarding how the encoder is handled during training:

1. The text initially states:"...we initialize the weights for the encoder... from the pre-trained VGGT model, and we keep the encoder frozen during training."

2. However, shortly after, when discussing the optimization and scheduler, it says:"...we assign different initial learning rates to model components: 5e-6 for the encoder..."

Could you please clarify whether the encoder is strictly frozen (parameters not updated) or if it is fine-tuned with this small learning rate?

Thank you! Best regard.
